modules/nf-core/amplify/predict/main.nf

```diff
@@ -2,7 +2,6 @@ process AMPLIFY_PREDICT {
     tag "$meta.id"
     label 'process_single'
 
-    // WARN: Version information not provided by tool on CLI. Please update version string below when bumping container versions.
```

Comment:
Did the tool gain a version output?

Reply:
looks like it....im not seeing the version being set anywhere explicitly.... @louperelo can confirm